setup.py

```diff
@@ -43,7 +43,7 @@ def load_requirements(*requirements_paths):
     return list(requirements)
 
 
-VERSION = '0.1.25'
+VERSION = '0.1.26'
```

Comment:
`GET` to `POST` and `POST` to `PUT` are breaking changes, and as per semantic versioning we should increase the MAJOR version which means `0.1.25` -> `1.1.25`